src/broadcast/listenbrainzlistener/listenbrainzjsonfactory.cpp

```diff
@@ -0,0 +1,38 @@
+
+#include <QDateTime>
+#include <QJsonArray>
+#include <QJsonDocument>
+#include <QJsonObject>
+
+#include "broadcast/listenbrainzlistener/listenbrainzjsonfactory.h"
+
+
+QByteArray ListenBrainzJSONFactory::getJSONFromTrack(TrackPointer pTrack, JsonType type) {
+    QJsonObject jsonObject;
+    QString stringType;
+    if (type == NowListening) {
+        stringType = "playing_now";
+    }
+    else {
+        stringType = "single";
+    }
+
+    QJsonArray payloadArray;
+    QJsonObject payloadObject;
+    QJsonObject metadataObject;
+    QString title = pTrack->getTitle();
+    QString artist = pTrack->getArtist();
+    metadataObject.insert("artist_name",artist);
```

Comment:
Mit Pick: move comma ...name", art...
The same below.

Reply:
Alright.